### PR TITLE
Adding note about non-Okta non-AD users for group password policy

### PR DESCRIPTION
**Pull request by jakubvul**
## Description:
- As in title. Also added "The" to beginning of paragraph that precedes note. 

### Resolves:

* [OKTA-184756](https://oktainc.atlassian.net/browse/OKTA-184756)

### CODE REVIEW

**Comment by cwu-okta** [comment]
@authteam-okta

File: _source/_docs/api/resources/policy.md [hunk 1]
```diff
@@ -1087,7 +1087,9 @@ The following conditions may be applied to the rules associated with MFA Enrollm
 ## Password Policy
 {: #GroupPasswordPolicy }
 
-Password policy controls settings that determine a user's password length and complexity, as well as the frequency with which a password can be changed.  This policy also governs the recovery operations that may be performed by the user, including change password, reset (forgot) password and self-service password unlock.
+The Password policy controls settings that determine a user's password length and complexity, as well as the frequency with which a password can be changed.  This policy also governs the recovery operations that may be performed by the user, including change password, reset (forgot) password and self-service password unlock.
```
Comment (wdawson-okta) [inline _source/_docs/api/resources/policy.md]: Small tweaks:

> The Password policy determines a user's password length and complexity, as well as the frequency with which a password must be changed.  This policy also governs the recovery operations that may be performed by the user, including change password, reset (forgot) password and self-service password unlock.

**Comment by wdawson-okta** [comment]
Thanks Jakub ⛵️

File: _source/_docs/api/resources/policy.md (hunk 1 above)
Comment (jakubvul) [reply _source/_docs/api/resources/policy.md]: Further tweaked to:

> The Password policy determines **the requirements for** a user's password length and complexity

**Pull request merged**